src/System.CodeDom/tests/System.CodeDom.Tests.csproj

```diff
@@ -82,6 +84,7 @@
     <Compile Include="CodeObjects\CodeObjectTestBase.cs" />
     <Compile Include="CodeStatements\CodeStatementTestBase.cs" />
     <Compile Include="CodeObjects\CodeChecksumPragmaTests.cs" />
+    <Compile Include="Microsoft\VisualBasic\VBCodeProviderTests.cs" />
```

Comment:
Nit: it looks like there are already some entries out of order, but please at least put the two Microsoft\ ones together.